src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -42,7 +42,7 @@ public function __construct(LoaderResolverInterface $resolver, array $defaultOpt
     /**
      * {@inheritdoc}
      */
-    public function load($resource, string $type = null)
+    public function load(mixed $resource, string $type = null)
```

Comment:
💡 Note: I can add `mixed` here without bumping the Config components to 6.0. `mixed` is the only parameter type that may be *added* downstream.